tests/unit/modules/test_btrfs.py

```diff
@@ -362,3 +362,300 @@ def test_properties_error(self):
         '''
         self.assertRaises(CommandExecutionError, btrfs.properties,
                           '/dev/sda1', 'subvol', True)
+
+    def test_subvolume_create(self):
+        '''
+        Test btrfs subvolume create
+        '''
+        # Fails when qgroupids is not a list
+        self.assertRaises(CommandExecutionError, btrfs.subvolume_create,
```

Comment:
Please use pytest instead. And for the raising exception should be another UT.

Reply:
Done

Reply:
@isbm What is the motivation for just suddenly switching to pytest here? We certainly have long-term plans to go in that direction but are we sure we want to just start importing pytest into individual tests like this?

Reply:
IMHO I do not like pytest. Basically because the magic that involves `assert`. In normal Python `assert` is a language statement that evaluate the expression (True or False) and throws an `AssertionError`.

Pytest change that to include the features that are already in the `self.assertEqual` family. Both code generate very similar report to show the differences of data structures. So I do not see the gain incorporating more magic to this problem.

But I am happy to adhere to the project expectations in regards to tests frameworks.

Reply:
> IMHO I do not like pytest.

`¯\_(ツ)_/¯`. This is what we are using here and pytest gives _better results_ to catch the error.

P.S. I like pytest.